config.json

```
@@ -485,6 +485,17 @@
                 "math",
                 "test_driven_development"
             ]
-        }
+         },
+        {
+            "slug": "prime-factors",
+            "uuid": "b6b7e95b-c4ca-438c-a5e7-d76cb44e5b19",
+            "core": false,
+            "unlocked_by": null,
+            "difficulty": 1,
+            "topics": [
+                "math",
+                "integers"
+            ]
+        },
```

```suggestion
        }
```

JSON doesn't allow trailing commas unfortunately